src/client/dfs/dfs.c

```diff
@@ -6479,3 +6478,246 @@ dfs_obj_anchor_set(dfs_obj_t *obj, uint32_t index, daos_anchor_t *anchor)
 
 	return daos_obj_anchor_set(obj->oh, index, anchor);
 }
+
+#define DFS_NR_ITERATE 128
+
+int
+oit_mark_cb(dfs_t *dfs, dfs_obj_t *parent, const char name[], void *arg)
+{
+	daos_handle_t	oit;
+	dfs_obj_t	*obj;
+	daos_obj_id_t	oid;
+	d_iov_t		marker;
+	bool		mark_data = true;
+	int		rc;
+
+	oit.cookie = (uint64_t)arg;
+
+	/** open the entry name and get the oid */
+	rc = dfs_lookup_rel(dfs, parent, name, O_RDONLY, &obj, NULL, NULL);
+	if (rc) {
+		D_ERROR("dfs_open() if %s failed: %d\n", name, rc);
```

Comment:
Agreed, the called function is dfs_lookup_rel() but the log says dfs_open(), this looks like a cut-and-paste error or similar.